### DISCUSSION TITLE
Where can I set up the GPT-4 model of revChatGPT.V1?

### DISCUSSION THREAD

**Discussion by mldljyh** [acheong08/ChatGPT#1189] [Q&A]
I can't find a way to set up the model using Python source code instead of 'python3 -m'. There is no information in the README or wiki, and even when I looked at the code for V1, there is no part where the user can set up the model, only code that retrieves the self model information. Where can I set up the model as GPT-4? I thought it was included in the config, but it wasn't.

src/revChatGPT/V1.py
            "model": self.config.get("model")
            or (
                "text-davinci-002-render-paid"
                if self.config.get("paid")
                else "text-davinci-002-render-sha"

**Comment by acheong08** [comment]
> Where can I set up the model as GPT-4? I thought it was included in the config, but it wasn't.

It is in the config. `"model": "gpt-4"`

**Reply by mldljyh** [reply]
I write "model": "gpt-4" in config. But error occurred.
code:
chatbot = Chatbot(config={
  "email": "id@example.com",
  "password": "password"
  "model": "gpt-4"
})

error:
    "model": "gpt-4"
           ^
SyntaxError: invalid syntax

**Reply by hrizal** [reply]
you need comma after password (maybe)

**Reply by acheong08** [reply]
> you need comma after password (maybe)

Correct.

@mldljyh I would suggest passing errors through ChatGPT before reporting. It could have spotted that

**Reply by mldljyh** [reply]
OMG. I didn't know I would make such a mistake. Thank you. @hrizal @acheong08

**Reply by henryyang1993** [reply]
can we fallback to the default gpt3.5 if gpt4 is not available or rate limited?
